Xamarin.Forms.Core/ListView.cs

```diff
@@ -121,11 +108,17 @@ protected override void OnBindingContextChanged()
 
 			object bc = BindingContext;
 
-			if (Header is Element header)
+			var header = Header as Element;
+			if (header != null)
+			{
 				SetChildInheritedBindingContext(header, bc);
+			}
```

Comment:
wasn't this good before ?

Reply:
Yea... All I did was rollback this file to before VT to ensure that it was rolled back properly

But I'll remove